charts/sonatype-nexus/templates/deployment-statefulset.yaml

```diff
@@ -81,6 +81,9 @@ spec:
               exec:
                 command: {{ .Values.deployment.postStart.command }}
           {{- end }}
+          {{- with .Values.nexus.envFrom }}
```

Comment:
Seems like you are missing the "envFrom:" prior to line 84

Reply:
:facepalm: ... Thanks